manifests/namespace-install/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../base
+  - ../crds
```

Comment:
kustomization.yaml files in the directories with your actual yaml manifests seems to make it no longer possible to just kubectl apply -f dir/, is this customary?